_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,407 @@
+
+The rule assumes that non-visual users will interpret some visual reference words as meaning "ahead" or "backwards" in the reading order. For example in most contexts "see the content below" will mean ahead in the reading order which is not a visual reference and should pass this test.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+### Understanding WCAG
+
+- [WCAG 2.1 - Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [WCAG 2.1 - Understanding Success Criterion 1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+
+### Related WCAG Techniques
+
+- [G96: Providing textual identification of items that otherwise rely only on sensory information to be understood](https://www.w3.org/WAI/WCAG21/Techniques/general/G96)
+- [F14: Failure of Success Criterion 1.3.3 due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+- [F26: Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information](https://www.w3.org/WAI/WCAG21/Techniques/failures/F26)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The content in the second column is indicated with the word "right" (which is an indicator based on visual perception) but also indicated by referencing the word "howdy".
+
+```html
+<head>
+	<title>Passed example 1 9bd38c</title>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+The button in the second column is indicated with the word "box" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order. Note that "below" is also a visual reference word but in this case it can also be accurately interpreted as "next in the DOM order" which does not rely on visual attributes alone.
+
+```html
+<div class="col-container">
+  <div class="col">
+    <p>Interact with the box below this paragraph, for a surprise</p>
+  </div>
+
+  <div class="col">
+    <button onclick="alert('Surprise!')">Howdy</button>
+  </div>
+</div>
+```
+
+#### Passed Example 3
```

Comment:
Because the situation is rather different. In one the word is included in the button itself. In the other the word is a heading above the content being referenced.